src/tools/cephfs/top/cephfs-top

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/python3
+
+import argparse
+import sys
+import curses
+import errno
+import json
+import signal
+import time
+
+from collections import OrderedDict
+from datetime import datetime
+from enum import Enum, unique
+
+import rados
+
+
+class FSTopException(Exception):
+    def __init__(self, msg=''):
+        self.error_msg = msg
+
+    def get_error_msg(self):
+        return self.error_msg
+
+
+@unique
+class MetricType(Enum):
+    METRIC_TYPE_NONE = 0
+    METRIC_TYPE_PERCENTAGE = 1
+    METRIC_TYPE_LATENCY = 2
+
+
+FS_TOP_PROG_STR = 'fstop'
+FS_TOP_PROG_VER = '0.0.1'
```

Comment:
Use ceph version.

Reply:
Hmmm... That means the version should be auto-updated (on new releases). Is there a standard way of doing that in ceph?

Reply:
and btw, this version is also recorded in `src/tools/cephfs/top/setup.py`